fundamentals/points.md

```diff
@@ -113,6 +154,88 @@ Individual points can each have different edge and face colors. You can initiall
 
 To change the point color properties from the GUI you must first select the points whose properties you want to change, otherwise you will just be initializing the property for the next point you add.
 
+## setting point edge and face color with properties
+Point edge and face colors can be set as a function of a property in `Points.properties`. There are two ways that the values in properties can be mapped to colors: (1) color cycles and (2) colormaps.
+
+Color cycles are sets of colors that are mapped to categorical properties. The colors are repeated if the number of unique property values is greater than the number of colors in the color cycle.
+
+Colormaps are a continuum of colors that are mapped to a continuous property value. The available colormaps are listed below (colormaps are from [vispy](http://vispy.org/color.html#vispy.color.Colormap)). For some guidance on choosing colormaps, see the [matplotlib colormap docs](https://matplotlib.org/3.2.0/tutorials/colors/colormaps.html).
+
+* autumn
+* blues
+* cool
+* greens
+* reds
+* spring
+* summer
+* fire
+* grays
+* hot
+* ice
+* winter
+* light_blues
+* orange
+* viridis
+* coolwarm
+* PuGr
+* GrBu
+* GrBu_d
+* RdBu
```

Comment:
Is this list accurate/exhaustive? Where are magma, inferno, plasma? Where is twilight? These certainly don't correspond to our image colormaps...

Reply:
They were used here https://napari.org/tutorials/fundamentals/image, we should probably correct them in both places or remove the full long list from these pages and link to a new page dedicated around our colormaps

Reply:
Yeah, that's where I pulled the list from. I can leave them as is and then we can make a separate PR with a page for colormaps.

Reply:
I made an issue here: https://github.com/napari/tutorials/issues/71